doc/book/reference/usage-examples.md

```diff
@@ -379,7 +384,7 @@ class Ping
 }
 ```
 
-Finally, in `public/index.php`, place the following:
+After that’s done run `composer dump-autoload` from the command-line, in the root directory of your project. Finally, in `public/index.php`, place the following:
```

Comment:
`composer dump-autoload` should be placed with different paragraph imo, so something like:

After that’s done, run:
```
 composer dump-autoload
```